libs/net/dal/Migrations/Initial/Up/PostUp/01-DataSource.sql

```diff
@@ -4003,4 +4008,6 @@ FROM (SELECT id FROM public.data_source WHERE public.data_source.code = CBC_CAPT
 WHERE public.data_source.code = CBC_CLIP_CODE;
 END IF;
 
+UPDATE public.data_source SET schedule_type = 1 where public.data_source.code = CASTANET;
```

Comment:
Don't do this update.  The script above should correct set the `schedule_type` for all data sources, there should be no reason to update it as this is the initial migration.

Reply:
Ah you didn't want to have to set the default value for every record.  This makes sense.